examples/tokenvm/config/config.go

```diff
@@ -101,6 +100,8 @@ func (c *Config) setDefault() {
 	c.MempoolVerifyBalances = defaultMempoolVerifyBalances
 	c.StateSyncServerDelay = c.Config.GetStateSyncServerDelay()
 	c.StreamingBacklogSize = c.Config.GetStreamingBacklogSize()
+	// TODO: hardcoded for testing, idk why gorilla doesn't like port 0.
+	c.StreamingPort = 4000
```

Comment:
I'll unify this with the default server so that we don't even have an extra port after this.

Reply:
If you want to make this change yourself, all you do is this: https://github.com/ava-labs/coreth/blob/61baf6dc01755513c195bd54c218b942d2d113d7/plugin/evm/vm.go#L1176-L1184

Reply:
Create one of these: https://github.com/ava-labs/hypersdk/blob/0907bf7c016c3ab569952201270e37cdfb8592b1/utils/utils.go#L35-L59

But call it `NewWebsocketHandler`. Change the name of the existing one to `NewRPCHandler`: https://github.com/ava-labs/hypersdk/blob/0907bf7c016c3ab569952201270e37cdfb8592b1/utils/utils.go#L35-L59

Reply:
Add it as an endpoint here under a new vm config called `vm.WebsocketEndpoint`: https://github.com/ava-labs/hypersdk/blob/0907bf7c016c3ab569952201270e37cdfb8592b1/examples/tokenvm/controller/controller.go#L128-L135

Then you can just use the standard port.